sympy/physics/mechanics/loads.py

```diff
@@ -0,0 +1,187 @@
+                        f"'{self.__class__.__name__}' and "
+                        f"'{other.__class__.__name__}'")
+
+    def __mul__(self, other):
+        raise TypeError(f"unsupported operand type(s) for *: "
+                        f"'{self.__class__.__name__}' and "
+                        f"'{other.__class__.__name__}'")
+
+    __radd__ = __add__
+    __rmul__ = __mul__
+
+
+class Force(LoadBase):
+    """Force acting upon a point.
+
+    Explanation
+    ===========
+
+    A force is a vector that is bound to a line of action. This class stores
+    both a point, which lies on the line of action, and the vector. A tuple can
+    also be used, with the location as the first entry and the vector as second
+    entry.
+
+    Examples
+    ========
+
+    A force of magnitude 2 along N.x acting on a point Po can be created as
+    follows:
+
+    >>> from sympy.physics.mechanics import Point, ReferenceFrame, Force
+    >>> N = ReferenceFrame('N')
+    >>> Po = Point('Po')
+    >>> Force(Po, 2 * N.x)
+    (Po, 2*N.x)
+
+    If a body is supplied, then the center of mass of that body is used.
+
+    >>> from sympy.physics.mechanics import Particle
+    >>> P = Particle('P', point=Po)
+    >>> Force(P, 2 * N.x)
+    (Po, 2*N.x)
+
+    """
+
+    def __new__(cls, point, force):
+        return super().__new__(cls, point, force)
+
+    def __repr__(self):
+        return f'Force(point={self.point}, force={self.force})'
+
+    @staticmethod
+    def _generate_location(point):
+        if isinstance(point, BodyBase):
+            return point.masscenter
+        if isinstance(point, Point):
+            return point
+        raise TypeError('Force location should of type Point.')
+
+    @staticmethod
+    def _generate_vector(force):
+        if isinstance(force, Vector):
+            return force
+        raise TypeError('Force vector should be of type Vector.')
+
+    @property
+    def point(self):
+        return self.location
+
+    @property
+    def force(self):
+        return self.vector
+
+
+class Torque(LoadBase):
+    """Force acting upon a point.
+
+    Explanation
+    ===========
+
+    A torque is a free vector that is acting on a reference frame, which is
+    associated with a rigid body. This class stores both the frame and the
+    vector. A tuple can also be used, with the location as the first entry and
+    the vector as second entry.
+
+    Examples
+    ========
+
+    A torque of magnitude 2 along N.x acting on a frame N can be created as
+    follows:
+
+    >>> from sympy.physics.mechanics import ReferenceFrame, Torque
+    >>> N = ReferenceFrame('N')
+    >>> Torque(N, 2 * N.x)
+    (N, 2*N.x)
+
+    If a body is supplied, then the frame fixed to that body is used.
+
+    >>> from sympy.physics.mechanics import RigidBody
+    >>> rb = RigidBody('rb', frame=N)
+    >>> Torque(rb, 2 * N.x)
+    (N, 2*N.x)
+
+    """
+
+    def __new__(cls, frame, torque):
+        return super().__new__(cls, frame, torque)
```

Comment:
Redundant?

Reply:
Understood. You're correct. It is a nice to have for the argument renaming.

Reply:
Well if you are using an IDE, it is quite nice to actually get the hint that the first argument is `frame` (not `location`) and the second is `torque`, which is clearer than `vector`